engine/cli/src/commands.rs

```diff
@@ -35,6 +35,9 @@ pub(crate) enum Commands {
 
     #[command(about = "Deploy a BAML project to Boundary Cloud")]
     Deploy(crate::deploy::DeployArgs),
+
+    #[command(about = "Format BAML source files")]
```

Comment:
```suggestion
    #[command(about = "Format BAML source files", name = "fmt", hide = true)]
```

I think we can merge this code as long as we hide this command from users